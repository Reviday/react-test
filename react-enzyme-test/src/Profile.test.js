import React from 'react';
import { mount } from 'enzyme';
import Profile from './Profile';

describe('<Profile />', () => {
  it('matches sanpshot', () => {
    // mount 함수는 Enzyme을 통하여 리액트 컴포넌트를 렌더링 해줍니다.
    const wrapper = mount(<Profile username="reviday" name="홍길동" />);
    expect(wrapper).toMatchSnapshot();
  });
});
