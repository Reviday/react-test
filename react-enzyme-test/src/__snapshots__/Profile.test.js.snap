// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> matches sanpshot 1`] = `
<Profile
  name="홍길동"
  username="reviday"
>
  <div>
    <b>
      reviday
    </b>
    <span>
      (
      홍길동
      )
    </span>
  </div>
</Profile>
`;
